scripts/build/.variables

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -eu
 
-VERSION=${VERSION:-"unknown-version"}
+VERSION=${VERSION:-$(cat VERSION)}
```

Comment:
This file could be called with a different working dir, so I think it needs to do something like this:

https://github.com/docker/cli/pull/343/files#diff-11460b30ebb8794f3c4f1061c483157dR4